### ISSUE TITLE
Enable ctrl-C for copy on all pages

### ISSUE THREAD

**Issue by ssbarsky** [jordanbrown0/reg3#476]
There are some places where the ctrl-C key works to copy text (like when in member edit mode) but other places it doesn't. I especially want to copy the text that a report generates and reformat it for my use (like a report by class that has many empty classes.)

This is also why I did screen shots for so many of my error reports or questions.

**Comment by jordanbrown0** [comment]
Yeah, I was just noticing that this weekend.  I think it works in editable fields, but not in other contexts.  I'm deliberately suppressing almost all control/alt/function operations, to make it harder for an incorrect keystroke to take you away from the application; most likely, that filtering just needs to be loosened up a little.

A workaround is to use the right-mouse-button menu or Edit/Copy.

**Comment by ssbarsky** [comment]
Thank you. right-mouse-buttton menu works.
